gems/pending/postgres_ha_admin/failover_monitor.rb

```diff
@@ -10,15 +10,20 @@ class FailoverMonitor
     FAILOVER_ATTEMPTS = 10
     DB_CHECK_FREQUENCY = 300
     FAILOVER_CHECK_FREQUENCY = 60
+    LOG_FILE = '/var/www/miq/vmdb/log/ha_admin.log'.freeze
     attr_accessor :failover_attempts, :db_check_frequency, :failover_check_frequency
 
     def initialize(db_yml_file: '/var/www/miq/vmdb/config/database.yml',
                    failover_yml_file: '/var/www/miq/vmdb/config/failover_databases.yml',
                    ha_admin_yml_file: '/var/www/miq/vmdb/config/ha_admin.yml',
-                   log_file: '/var/www/miq/vmdb/log/ha_admin.log',
+                   logger: nil,
```

Comment:
I think all of these are compatible with the way this is currently written.

For example, the specs use a few of the forms you mention [here](https://github.com/ManageIQ/manageiq/pull/10558/files#diff-69e88ff9be55ed958c33c81c9e643528R21) and [here](https://github.com/ManageIQ/manageiq/pull/10558/files#diff-69e88ff9be55ed958c33c81c9e643528R39)